scripts/unique-kmers.py

```diff
@@ -66,16 +87,20 @@ def get_parser():
                         help='Acceptable error rate')
 
     parser.add_argument('-R', '--report',
```

Comment:
If you put the long option first then the usage summary line is more readable:

Currently it is
```
unique-kmers.py [-h] [--version] [-q] [--ksize KSIZE] [--error-rate ERROR_RATE] [-R filename] [--stream-out] [--diagnostics] input_sequence_filename [input_sequence_filename ...]

```